_posts/07-05-01-Abstraction-Layers.md

```diff
@@ -0,0 +1,29 @@
+---
+isChild: true
+title: Slojevi za apstrakciju (Database Abstraction Layers - DAL)
+anchor: databases_abstraction_layers
+---
+
+## Slojevi za apstrakciju (Database Abstraction Layers - DAL) {#databases_abstraction_layers_title}
+
+Mnogi framework-ovi imaju sopstveni DAL koji može, a ne mora da bude "nad" [PDO-om][1].
+Oni najčešće emuliraju mogućnosti jednog sistema baze podataka koje nedostaju u drugom, tako što
```

Comment:
Hm, nekako mi se više sviđa baš emuliraju nego imitiraju. Možda da stavim *emuliraju/oponašaju* ili tako nešto?